grafana-plugin/src/network/oncall-api/types-generator/custom-schemas.ts

```diff
@@ -1,3 +1,7 @@
 // Custom properties not exposed by OpenAPI schema should be defined here
 
-export type CustomApiSchemas = {};
+export type CustomApiSchemas = {
+  AlertReceiveChannel: {
+    deleted?: boolean;
```

Comment:
@vadimkerr seems that we're using this property on the frontend but it's not exposed in openapi schema